Formula/charm-tools.rb

```diff
@@ -24,29 +24,37 @@ class CharmTools < Formula
     sha256 "caabb9c22961a3413ac85cd1e5525ec9ca80daeba6555f4f60802b6c256e252b"
   end
 
+  # pyyaml>=3.11,<4.3
```

Comment:
What's with the comments?

Reply:
We usually use the latest version available on pypi unless the package forces a specific version or range of versions. If we don't use the latest version, we typically add a comment pointing the package restrictions.

Reply:
Can/have we documented that somewhere?

Reply:
that is just my note (grabbed from setup.py, https://github.com/juju/charm-tools/blob/master/setup.py#L47)